### DISCUSSION TITLE
Arguments missing for parameters "receive", "send" Pylance reportCallIssue

### DISCUSSION THREAD

**Discussion by tsanyqudsi** [dymmond/esmerald#395] [General]
In VSCode, I get an error from `pylance` when I use `@get()`. Is there any way that I can avoid or fix that problem ?

The error came up when i do something like this :
```python
from esmerald import get


@get()
async def get() -> str:
    return "Hello, World!"
```

**Comment by tarsil** [comment]
> In VSCode, I get an error from `pylance` when I use `@get()`. Is there any way that I can avoid or fix that problem ?
> 
> The error came up when i do something like this :
> 
> ```python
> from esmerald import get
> 
> 
> @get()
> async def get() -> str:
>     return "Hello, World!"
> ```

Hi. Error? Like red line below? Do you have a screenshot? Maybe you are not pointing to your virtual environment where Esmerald is installed?

**Reply by tarsil** [reply]
Also I don't understand the title and then just what you asked. Is all the same?
